tethys_gizmos/gizmo_options/jobs_table.py

```diff
@@ -117,44 +330,64 @@ def __init__(self, jobs, column_fields, show_status=True, show_actions=True,
         self.enable_data_table = enable_data_table
         self.data_table_options = data_table_options or {'ordering': True, 'searching': False, 'paging': False}
 
-        actions = actions or ['run', 'resubmit', 'logs', 'terminate', 'delete']
-        if monitor_url:
-            actions.append('monitor')
-        if results_url:
-            actions.append('results')
-
-        # code for backwards compatibility. Remove in Tethys v3.3
-        if status_actions is not None:
-            # deprecation warning
-            write_warning('Deprecation Warning: The "status_actions" option in JobsTable will be removed in '
-                          'the next release of Tethys. Please use "show_status" and "show_actions" instead.')
-            self.show_actions = status_actions
-        lcl = locals()
-        for option, action in (('run_btn', 'run'), ('delete_btn', 'delete'), ('show_resubmit_btn', 'resubmit'),
-                               ('show_log_btn', 'logs')):
-            option_val = lcl[option]
-            if option_val is not None:
-                # deprecation warning
-                write_warning(f'Deprecation Warning: The "{option}" option in JobsTable will be removed in the '
-                              f'next release of Tethys. Please use the "show_actions" and "actions" options instead.')
-                if option_val:
-                    actions.append(action)
-                else:
-                    try:
+        actions = actions or ['run', 'resubmit', '|', 'logs', 'monitor', 'results', '|', 'terminate', 'delete']
+
+        # TODO: this is backward compatibility code, remove at next major release
+        if '|' not in actions:
+            # revert to previous default order
+            actions.extend(['monitor', 'results'])
+            actions = set(actions)
+            ordered_actions = list()
+            prev_len = 0
+            for action_group in [
+                ['run', 'pause', 'resume', 'resubmit'],
+                ['logs', 'monitor', 'results'],
+                ['terminate', 'delete'],
+            ]:
+                for action in action_group:
+                    if action in actions:
+                        ordered_actions.append(action)
                         actions.remove(action)
-                    except ValueError:
-                        pass
+                if len(ordered_actions) > prev_len:
+                    ordered_actions.append('|')
+                prev_len = len(ordered_actions)
+            ordered_actions.extend(actions)
+            actions = ordered_actions[:-1] if ordered_actions[-1] == '|' else ordered_actions
         # end compatibility code
```

Comment:
Previously you could provide a list of any of the provided actions in any order and it would always display them in the same order with the dividers in designated places.

For example, previously you could specify `actions` like this:

```python
actions=[ 'log', 'monitor', 'resubmit', 'terminate', 'run', 'delete', 'results'],
```

And it would render a dropdown like this:

```
Run
Resubmit
----------
View Logs
Monitor Job
View Results
-------------
Terminate
Delete
```

With that same input it would now render it like this:

```
View Logs
Monitor Job
Resubmit
Terminate
Run
Delete
View Results
```

Reply:
The functionality should still be the same, so maybe it's not worth keep the compatibility code at this point.

Reply:
Ii ended up removing it.